wp-content/themes/workingnyc/views/programs/archive.vue

```diff
@@ -185,14 +185,13 @@
                     <span v-if="query.page==headers.pages" class="pb-[10px] pl-[4px] pr-[4px] text-[14px] border-b-4">{{headers.pages}}</span>
                     <span v-else class="pb-[10px] pl-[4px] pr-[4px] text-[14px]">{{headers.pages}}</span>
                   </button>
-                  <button v-if="next" class="text-[#080707] font-normal" @click="nextPagination">
+                  <button v-if="next" class="text-[#080707] font-normal" @click="nextPage">
                     <svg class="h-3 w-3 fill-none stroke-black">
                       <use href="#lucide-chevron-right"></use>
                     </svg>
                   </button>
                 </div>
                 
-                <article class="c-alert mb-3" data-js="alert-help" v-else-if="strings.SUGGEST" v-html="strings.SUGGEST"></article>
```

Comment:
I deleted some code that seemed to be unused and added comments, let me know if these are fine with you https://github.com/CityOfNewYork/working-nyc/pull/292/commits/13423248973ff1151d78940e941b396671faf288